apmapf.py

```diff
@@ -28,7 +29,10 @@
 import math
 import logging
 import timeout_decorator
-sys.stderr = open(os.devnull, 'w')
+
+# We don't care about errors thrown during import of pysmt.*.
```

Comment:
Not sure if it's worth including this comment and the other one on line 50.

Reply:
Good comment to leave -- the pysmt upstream had some deprecation warnings that I didn't want to keep looking at, so I disabled the errors... Perhaps I shouldn't have done that, and in future we can just remove these lines